### PR TITLE
[ADD] website_hr_recruitment_legal: Legal stuff for job recruitment

### PR DESCRIPTION
**Pull request by yajo**
So you ask for people's data without legal terms? Bad boy... Here's the fix.

@Tecnativa

### CODE REVIEW

**Comment by martintorresvm** [comment]
@Yajo I tested on runbot and there is two things could be wrong.

1. I had to install manually `website_hr_recruitment` and configure to post positions on the website (some instructions about setup would be necessary)
2. I could not find the checkbox no matter I do

website_hr_recruitment_legal

**Comment by yajo** [comment]
Right! Dependency was missing. Please review.

File: website_hr_recruitment_legal/templates/apply.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+
+    <template id="apply"
+            inherit_id="website_hr_recruitment.apply"
+            name="Add legal acceptance checkbox">
+        <xpath expr="//*[hasclass('form-group')][//*[hasclass('o_website_form_send')]"
```
Comment (Tardo) [inline website_hr_recruitment_legal/templates/apply.xml]: Typo... you have an extra square bracket... 

```suggestion
        <xpath expr="//*[hasclass('form-group')]//*[hasclass('o_website_form_send')]"
```

**Review by Tardo — CHANGES_REQUESTED** [review]

File: website_hr_recruitment_legal/templates/apply.xml (hunk 1 above)
Comment (yajo) [reply website_hr_recruitment_legal/templates/apply.xml]: Thanks! Actually I had one less :grin:

**Review by Tardo — APPROVED** [review]
LGTM, tested on runbot

**Comment by martintorresvm** [comment]
Not sure what I am doing wrong but I could not find the checkbox which is means to accept terms

**Comment by Tardo** [comment]
@victormmtorres can't see this?
screenshot_20190603-153343_1920x1080

**Comment by martintorresvm** [comment]
My bad, I was expecting at some other part like after filling all fields. All good then

**Comment by yajo** [comment]
@victormmtorres Please use [the approve tool in github](https://help.github.com/en/articles/about-pull-request-reviews). This helps the @OCA-git-bot know whether this should be merged.

**Review by martintorresvm — APPROVED** [review]

**Pull request merged**